backend/repos/query.go

```diff
@@ -0,0 +1,142 @@
+package repos
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/google/uuid"
+	"github.com/honeynet/ochi/backend/entities"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/jmoiron/sqlx/reflectx"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type QueryRepo struct {
```

Comment:
Everything exported (capitalized) needs a doc string starting with the exported name:
```go
// Exported function
func Exported()
```

Reply:
Done